ui/orbital.c

```diff
@@ -535,24 +539,17 @@ static void* orbital_display_main(void* arg)
         ORBITAL_WIDTH, ORBITAL_HEIGHT, flags);
 
     count = 0;
-    if (!SDL_Vulkan_GetInstanceExtensions(ui.sdl_window, &count, NULL)) {
-        error_report("SDL_Vulkan_GetInstanceExtensions returned false");
-        return NULL;
-    }
+    SDL_Vulkan_GetInstanceExtensions(ui.sdl_window, &count, NULL);
```

Comment:
What's the reasoning behind this change? `SDL_Vulkan_GetInstanceExtensions` can fail, and whether it does is returned in an `SDL_bool` value [1]. We should check for that, and if it returns `SDL_FALSE` find the root issue and fix it.

[1] https://wiki.libsdl.org/SDL_Vulkan_GetInstanceExtensions